lib/organisms/gravity-form/example.html

```diff
@@ -0,0 +1 @@
+<ln-o-gravity-form id="{{id}}" token="{{token}}" data="{{data}}"></ln-o-gravity-form>
```

Comment:
We can load it in the "static" api. So we can inject the relevant service (lnCmsService I think it's called) into the GF Service and get it that way. We should set up caching on the $http service so it won't make another call to the server (there's a task open for this on the ln-cms project already). Anyway, this is fine for now, we can refactor later.